### ISSUE TITLE
wasm-pack build flag docs out of date

### ISSUE THREAD

**Issue by twilco** [rustwasm/wasm-pack#462]
## 🐛 Bug description
https://rustwasm.github.io/wasm-pack/book/commands/build.html#profile

The section of this book describes three flags: `--dev`, `--profiling`, and `--release`, all of which result in this errored output:

```
$ wasm-pack build --release  
error: Found argument '--release' which wasn't expected, or isn't valid in this context

USAGE:
    wasm-pack build [FLAGS] [OPTIONS] [path]

For more information try --help
```

It looks like `--debug` is the only flag that does anything currently - no flag seems to specify build for release by default.

#### 🤔 Expected Behavior
The documentation matches the behavior of the tool.

#### 👟 Steps to reproduce
Run `wasm-pack` with each command and note the output.

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.5.1
rustc version: rustc 1.31.0 (abe02cefd 2018-12-04)

Thanks for your work on this awesome tool!

**Comment by drager** [comment]
Yeah, the docs are built from `master` and that behavior was added in #440 which targets the `0.6.0` milestone which hasn't been released yet. This means that `0.5.1` doesn't have this feature.

I think the docs at https://rustwasm.github.io/wasm-pack/book/ should be generated for multiple versions (like docs.rs does), so we can have docs for `master`, `latest stable` as well as older releases.

**Comment by twilco** [comment]
Got it, thanks.  Any idea when `0.6.0` will be coming?

**Comment by drager** [comment]
Not really, sorry. I think @ashleygwilliams should be able to answer that though.

**Comment by ashleygwilliams** [comment]
I will be working on a release starting tomorrow, so should be sometime this week! So sorry for the delay! The Rust 2018 Edition kinda took over a bunch of my time. Now that that is iver you should see both dev and release work in this pick up sognificantly. Sorry again!
